deploy/templates/flexible-azuredeploy.json

```diff
@@ -1,881 +1,881 @@
-        "attributes": {
-          "enabled": true
-        }
-      }
-    },
-    {
-      "type": "Microsoft.KeyVault/vaults/secrets",
-      "apiVersion": "2016-10-01",
-      "name": "[concat(variables('key_vault_name'), '/fhirserver-authority')]",
-      "location": "[parameters('ResourceLocation')]",
-      "dependsOn": [
-        "[resourceId('Microsoft.KeyVault/vaults', variables('key_vault_name'))]"
-      ],
-      "properties": {
-        "contentType": "text/plain",
-        "value": "[parameters('FhirServiceAuthority')]",
-        "attributes": {
-          "enabled": true
-        }
-      }
-    },
-    {
-      "type": "Microsoft.KeyVault/vaults/secrets",
-      "apiVersion": "2016-10-01",
-      "name": "[concat(variables('key_vault_name'), '/fhirserver-clientid')]",
-      "location": "[parameters('ResourceLocation')]",
-      "dependsOn": [
-        "[resourceId('Microsoft.KeyVault/vaults', variables('key_vault_name'))]"
-      ],
-      "properties": {
-        "contentType": "text/plain",
-        "value": "[parameters('FhirServiceClientId')]",
-        "attributes": {
-          "enabled": true
-        }
-      }
-    },
-    {
-      "type": "Microsoft.KeyVault/vaults/secrets",
-      "apiVersion": "2016-10-01",
-      "name": "[concat(variables('key_vault_name'), '/fhirserver-clientsecret')]",
-      "location": "[parameters('ResourceLocation')]",
-      "dependsOn": [
-        "[resourceId('Microsoft.KeyVault/vaults', variables('key_vault_name'))]"
-      ],
-      "properties": {
-        "contentType": "text/plain",
-        "value": "[parameters('FhirServiceClientSecret')]",
-        "attributes": {
-          "enabled": true
-        }
-      }
-    },
-    {
-      "type": "Microsoft.KeyVault/vaults/secrets",
-      "apiVersion": "2016-10-01",
-      "name": "[concat(variables('key_vault_name'), '/fhirserver-resource')]",
-      "location": "[parameters('ResourceLocation')]",
-      "dependsOn": [
-        "[resourceId('Microsoft.KeyVault/vaults', variables('key_vault_name'))]"
-      ],
-      "properties": {
-        "contentType": "text/plain",
-        "value": "[parameters('FhirServiceResource')]",
-        "attributes": {
-          "enabled": true
-        }
-      }
-    },
-    {
-      "type": "Microsoft.KeyVault/vaults/secrets",
-      "apiVersion": "2016-10-01",
-      "name": "[concat(variables('key_vault_name'), '/fhirserver-url')]",
-      "location": "[parameters('ResourceLocation')]",
-      "dependsOn": [
-        "[resourceId('Microsoft.KeyVault/vaults', variables('key_vault_name'))]"
-      ],
-      "properties": {
-        "contentType": "text/plain",
-        "value": "[parameters('FhirServiceUrl')]",
-        "attributes": {
-          "enabled": true
-        }
-      }
-    }
-  ],
-  "outputs": {
-  }
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
>"http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#", [](http://example.com/codeflow?start=13&length=81)

Just to verify, this is a rename of the old default template?

Reply:
Yes, given that in the ReadMe, we have this listed:
Open Source Deployment - Flexible: Describes how to deploy the IoMT FHIR Connector for Azure using another FHIR server or identity provider. 
Decided to let this be instead of removing it, and rename it instead.